### PR TITLE
Huda

### PR DESCRIPTION
**Pull request by kpslp**
@SenthilKumarSivakami

### CODE REVIEW

File: policy-as-code/revision-history.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Custom policy as code (PAC) with AWS Security hub
+
+# Revision History
```
Comment (SenthilKumarSivakami) [inline policy-as-code/revision-history.md:3]: please remove this section and the following table.

File: policy-as-code/overview.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+| **No. of Occurrences** | **Cost Calculation** | **Total Cost** |
+| --- | --- | --- |
+| **60 email notifications** | 60 x 1 region x $0.00 per notification (free tier: first 1,000 notifications) | $0.00 |
+| | Total SNS cost per month (USD) | $0.00 |
+
+*(Optional: Agency responsibility)
+
+> **Note:**
+> - For more information, refer to the links provided in Section 6: References
```
Comment (SenthilKumarSivakami) [inline policy-as-code/overview.md:93]: please remove this note in line 93. retain the below note. 

The links are listed under the Security Hub findings retention period. Hazel suggested to move this as section 2.3 (w.r.t the word doc) but i think we can call this as a separate md file  title "Security Hub findings", I don't have the full picture now.

**Review by SenthilKumarSivakami — CHANGES_REQUESTED** [review]

File: policy-as-code/_sidebar.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+- **Policy as code**
+  - [Revision history](policy-as-code/revision-history)
```
Comment (SenthilKumarSivakami) [inline policy-as-code/_sidebar.md]: Need to remove this.

**Comment by kpslp** [comment]
hi @SenthilKumarSivakami, changes made

File: policy-as-code/_sidebar.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+  - [GCC 2.0 Policy as code overview](policy-as-code/overview)
```
Comment (SenthilKumarSivakami) [inline policy-as-code/_sidebar.md:2]: can be just Overview , you may also refer to other docs (SEED , GCC or Ship-hats )

File: policy-as-code/_sidebar.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+  - [PaC rules](policy-as-code/pac-rules)
```
Comment (SenthilKumarSivakami) [inline policy-as-code/_sidebar.md:3]: the pac-rules.md file is blank, should there be any content here? if not, remove this from the sidebar for now.

File: policy-as-code/overview.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
+# Policy as Code - Overview
+
+## Audience
+
+- Agency Chief Information Security Officer (ACISO)
+- Agency Security Incident Response Officer (ASIRO)
+- Agency account administrators.
```
Comment (SenthilKumarSivakami) [inline policy-as-code/overview.md:7]: should be Agency Account Administrators.

**Review by SenthilKumarSivakami — CHANGES_REQUESTED** [review]

File: policy-as-code/overview.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
+# 1. Policy as Code - Overview
```
[the remark this answers is not in the document]
Comment (kpslp) [reply policy-as-code/overview.md]: removed. refer to revised version. removed for all section numbering except for GT xx

File: policy-as-code/overview.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+
+## 1.1. Audience
+
+This playbook is intended for the following agency users:
```
[the remark this answers is not in the document]
Comment (kpslp) [reply policy-as-code/overview.md]: removed

File: policy-as-code/revision-history.md (hunk 1 above)
Comment (kpslp) [reply policy-as-code/revision-history.md]: removed

File: policy-as-code/overview.md (hunk 2 above)
Comment (kpslp) [reply policy-as-code/overview.md]: moved to Security Hub section

File: policy-as-code/_sidebar.md (hunk 3 above)
Comment (kpslp) [reply policy-as-code/_sidebar.md]: removed

File: policy-as-code/_sidebar.md (hunk 4 above)
Comment (kpslp) [reply policy-as-code/_sidebar.md]: changed

File: policy-as-code/_sidebar.md (hunk 5 above)
Comment (kpslp) [reply policy-as-code/_sidebar.md:3]: as mentioned, large diff, will have to preview on staging

File: policy-as-code/overview.md (hunk 6 above)
Comment (kpslp) [reply policy-as-code/overview.md:7]: changed

File: policy-as-code/overview.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (SenthilKumarSivakami) [reply policy-as-code/overview.md]: Thanks!

File: policy-as-code/security-hub-findings-retention-period.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
+# Security Hub findings retention period
```
Comment (SenthilKumarSivakami) [inline policy-as-code/security-hub-findings-retention-period.md:1]: This section talks about things other than the retention period, right? Can we modify the heading to "Security Hub findings"?

File: policy-as-code/_sidebar.md [hunk 10]
```diff
@@ -0,0 +1,6 @@
+- **Policy as code**
+  - [Overview](policy-as-code/overview)
+  - [PaC rules](policy-as-code/pac-rules)
+  - [security-hub-findings-retention-period](/policy-as-code/security-hub-findings-retention-period)
```
Comment (SenthilKumarSivakami) [inline policy-as-code/_sidebar.md:4]: should we specifically say that it is the retention period if the section talks about other info too?

**Pull request merged**